rubocop/fork_usage.rb

```diff
@@ -33,7 +33,7 @@ def on_if(node)
       def on_send(node)
         return unless bad_fork(node)
         return if self.good_nodes.include?(node)
-        add_offense(node, :expression, MSG)
+        add_offense(node, location: :expression, message: MSG)
```

Comment:
`add_offsense` in rubocop no longer support positional arguments and instead, we need to use keyword arguments.